maven-core/src/main/java/org/apache/maven/plugin/internal/ValidatingConfigurationListener.java

```diff
@@ -41,12 +50,20 @@
 
     private final ConfigurationListener delegate;
 
+    private final MojoDescriptor mojoDescriptor;
+    private final ExpressionEvaluator expressionEvaluator;
+
+    private final Logger logger = LoggerFactory.getLogger( getClass() );
```

Comment:
Why not use a static logger, any reason to have a per-instance?

Reply:
No reason. As I see it is mixed in code. I didn't analize it.